scram/route_manager/tests/test_websockets.py

```diff
@@ -80,9 +80,9 @@ def setUp(self):
         self.generate_add_msgs = [lambda ip, mask: {"type": "translator_add", "message": {"route": f"{ip}/{mask}"}}]
 
         # Now we run any local setup actions by the child classes
-        self.local_setUp()
+        self.local_setup()
```

Comment:
This is a django-ism, changing it might confuse Django progammers.

Reply:
`setUp` is, and it's exempted from this check, but `local_setUp` is our own invention. Only 2 other repos in GitHub use it.